modules/distribution/src/repository/resources/conf/templates/repository/conf/tomcat/catalina-server.xml.j2

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<Server port="8005" shutdown="SHUTDOWN" xmlns:svns="http://org.wso2.securevault/configuration">
+
+    <Service className="org.wso2.carbon.tomcat.ext.service.ExtendedStandardService" name="Catalina">
+
+        <!--
+        optional attributes:
+
+        proxyPort="80"
+        -->
+        <Connector protocol="org.apache.coyote.http11.Http11NioProtocol"
+        {% for property_name,property_value in transport.http.items() %}
+                   {{property_name}}="{{property_value}}"
+                           {% endfor %}
+        />
+
+        <!--
+        optional attributes:
+
+        proxyPort="443"
+        Added sslEnabledProtocols="TLSv1,TLSv1.1,TLSv1.2" for poodle vulnerability fix
+        -->
```

Comment:
Fix comments indentation

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059